presto-main/src/test/java/com/facebook/presto/sql/analyzer/TestAnalyzer.java

```diff
@@ -836,6 +836,22 @@ public void testInvalidInsert()
                 "INSERT INTO t6 (a, b, c, d) VALUES (1, 1, 1)");
     }
 
+    @Test
+    public void testInvalidInsertNested()
+    {
+        assertFails(MISMATCHED_SET_COLUMN_TYPES,
+                "line 1:29: Mismatch at column 2: 'b.x.z' is of type double but expression is of type varchar.3.",
```

Comment:
This might be out of the scope of this PR, but `varchar.3` seems a bit weird 😂. Ideally it should be `varchar(3)`

Reply:
it actually prints out varchar(3) but for the expected string i believe we are using dot as a wildcard to match the (